db/mappings/model.py

```diff
@@ -20,22 +20,31 @@ class Status(enum.Enum):
     FAILED = "failed"
 
 
+class Site(enum.Enum):
```

Comment:
I'm sad that we have to manage this duplicate info across all the different repos... I wonder if there's some kind of config service (maybe AWS?) that could be useful here. (Or, it may be more trouble than it's worth)